### ISSUE TITLE
Not processing text containing html tags properly

### ISSUE THREAD

**Issue by cfx1** [LeaVerou/awesomplete#16932]
Adding an item containing html tags to the list, such as the text: `Text escaping: <xyz>tag</xyz>` will yield incorrect suggestions.
### Setup

```
<input id="myinput" />
<script>
    var input = document.getElementById("myinput");
    new Awesomplete(input, {
        minChars: 1,
        list: ["Text escaping: <xyz>tag</xyz>"]
    });
</script>
```

or

```
<input class="awesomplete" data-list="#mylist" data-minchars="1" />
<ul id="mylist">
    <li>Text escaping: &lt;xyz&gt;tag&lt;/xyz&gt;</li>
</ul>
```
### Results

Typing the letter 't' in the input box will give the following incorrect suggestion (missing the `<xyz>` and `</xyz>` that was surrounding the word 'tag'):
![image](https://cloud.githubusercontent.com/assets/9679270/17728053/044d46b4-6412-11e6-97f0-f86b4df76f5b.png)

Typing the letter 'x' in the input box will give the following incorrect suggestion (notice the missing '<' after the word "tag" in the suggestion):
![image](https://cloud.githubusercontent.com/assets/9679270/17728102/5e98777e-6412-11e6-9201-a0ebbbfc2b61.png)

**Comment by ArthurHoaro** [comment]
I have a similar issue. It looks like awesomplete interprets HTML within suggestions.
Is there a setting to prevent this behavior and keep the tags escaped properly?

Note that `<script>` tags are properly escaped.

**Comment by LeaVerou** [comment]
I don't believe so. I would be willing to merge a PR on this, since it can also be a security issue.
